src/odemis/gui/conf/data.py

```diff
@@ -594,6 +644,27 @@
             ("brightness", {
                 "control_type": odemis.gui.CONTROL_SLIDER,
             }),
+            ("mode", {
+                "label": "Detector Mode",
+            }),
+            ("type", {
+                "label": "Detector Type",
+            }),
+        )),
+    "se-detector-ion":
+        OrderedDict((
+            ("brightness", {
+                "label": "Brightness",
+            }),
+            ("contrast", {
+                "label": "Contrast",
```

Comment:
is it not necessary to have a control type here?

Reply:
updated, and added tooltips